lib/src/smooth_counter.dart

```diff
@@ -130,6 +141,8 @@ class _SmoothCounterState extends State<SmoothCounter> {
                 controller.duration,
             curve: widget.controller?.curve ?? widget.curve ?? controller.curve,
             controller: controller,
+            prefix: widget.prefix,
+            suffix: widget.suffix,
```

Comment:
Instead of passing the prefix and the suffix in this and displaying it, it is smoother to add Row here and display it following for example:

```dart
return IgnorePointer(
  child: Row( // <- add this
    mainAxisSize: MainAxisSize.min,
    children: [
      if (widget.prefix != null)
        Text(widget.prefix!, style: widget.textStyle),
      AnimatedSize(
        // ...
      ),
      if (widget.suffix != null)
        Text(widget.suffix!, style: widget.textStyle),
    ],
  ),
);
```

Reply:
Good catch! Thanks for the feedback :>. I've made the changes according to your suggestion.